pvp/common/message.py

```diff
@@ -144,22 +142,23 @@ def __init__(self,
                 this attr, when present, specifies which is being set.
         """
         if isinstance(name, str):
-            try:
-                name = values.CONTROL.__members__[name]
-            except KeyError as e:
+            ls = [x for x in values.CONTROL if str(x) == name]
+            if len(ls)>0:
+                name = ls[0]
+            else:
                 logger = init_logger(__name__)
                 logger.exception(f'Couldnt create ControlSetting with name {name}, not in values.CONTROL')
-                raise e
```

Comment:
also not sure about this change, the check seems equivalent and i don't know why pytest stuff is in the main codebase?

Reply:
hmm, yeah. For the first thing, `.__members__[name]` doesn't exist. The second,  we now explicitly test this exception. This is to warn pytest that an exception will come